docs/changelog.md

```diff
@@ -1,5 +1,11 @@
 # TBD
 
+### Changes
+* Upgrade Core to 1.58.0
+### Breaking Changes
+* Upgraded Core to 1.58.0 - which archives the contents of the directory in the root of the archive without additional nesting
```

Comment:
Can we add more context here? Just "which archives the contents of the directory in the root of the archive without additional nesting" I think would be helpful with extra language about what this refers to, etc.

A good heuristic for changelog entries is, "imagine that a user is reading this" - we want to make sure they understand what's going on, why, and what they need to do. (And this is because changelog entries are actually exposed to users on the docs page)